eng/scripts/Create-ApiReview.ps1

```diff
@@ -22,7 +22,6 @@ Param (
   [string] $Language
 )
 
-Set-StrictMode -Version 3
```

Comment:
I'm OK removing this to unblock things but the reason we enable it is to help find some of these issues so we should see about re-enabling it once we fix any outstanding issues.